saleor/graphql/schema.graphql

```diff
@@ -2265,10 +2265,10 @@ type Mutation {
   checkoutShippingAddressUpdate(checkoutId: ID!, shippingAddress: AddressInput!): CheckoutShippingAddressUpdate
   checkoutShippingMethodUpdate(checkoutId: ID, shippingMethodId: ID!): CheckoutShippingMethodUpdate
   checkoutUpdateVoucher(checkoutId: ID!, voucherCode: String): CheckoutUpdateVoucher
-  checkoutUpdateMetadata(id: ID!, input: MetaInput!): CheckoutUpdateMeta
-  checkoutClearMetadata(id: ID!, input: MetaPath!): CheckoutClearMeta
-  checkoutUpdatePrivateMetadata(id: ID!, input: MetaInput!): CheckoutUpdatePrivateMeta
-  checkoutClearPrivateMetadata(id: ID!, input: MetaPath!): CheckoutClearPrivateMeta
+  checkoutUpdateMetadata(input: MetaInput!, token: UUID!): CheckoutUpdateMeta
+  checkoutClearMetadata(input: MetaPath!, token: UUID!): CheckoutClearMeta
+  checkoutUpdatePrivateMeta(id: ID!, input: MetaInput!): CheckoutUpdatePrivateMeta
```

Comment:
- I don't see changes that modified these mutations.

- Also, we can't modify/remove mutations/queries between releases without marking them as deprecated first.

Reply:
- "I don't see changes that modified these mutations." - Is the schema auto-generated from the code? I don't quite understand this

- Gotcha! So to clarify, do I create the new ones with a different name, then add the deprecated flag to the old ones?

Reply:
I've changed the schema back to what it was before here, I changed it to match orders, but it's probably best to leave it

Reply:
`Is the schema auto-generated from the code?` -> npm run build-schema
You can leave it for now. In the next weeks, we will work on meta mutations

Reply:
Ahaaa gotcha!